### PR TITLE
UI Overhaul

### PR DESCRIPTION
**Pull request by rasyidf**
Update Layout and Flow, Fix Included

- Update Responsiveness
- Simplify Side Navigation to only show main navigation
- Wide Screen Layout Support
- Tiny bit of Animation
- UI Restructuration
- Improve User Experience in Date Range Selector
- Dashboard Design Update
- Tabify Settings.

### CODE REVIEW

**Review by rasyidf — COMMENTED** [review]
I've checked 2 times, there are no typos, and no breaking change while running the program, changes on visualization is untested because of lack of data.

**Comment by fahminlb33** [comment]
Here are a few noticable incositency from existing code and this PR changes.

* Sensors > Statistics, the button is too large
* Forecast (ML), the chart should be taller to allow more precise visualization
* Users, the button for admin is grayed out but still clickable, is the the desired changes? (might be related to #8)
* Settings, please remove the whitespace after status text. Punctuation should come immediately after the text without whitespace.
* About, maybe we should add out LinkedIn too, just in case. Also, would it be possible to use icons instead of text links?

File: Views/Home/Index.cshtml [hunk 1]
```diff
@@ -64,34 +62,124 @@
...
+                </div>
+            </div>
 
+        </div>
 
+        <div class="col-xxl-4 col-xl-6 col-lg-6 col-12">
+            <div class="card mb-4">
+                <div class="card-header">Venjix Information</div>
+                <div class="card-body">
+                    Version : 0.0.0.0
```
Comment (fahminlb33) [inline Views/Home/Index.cshtml]: Version should be retrieved from static field from code, maybe using Reflection or another static field so this value will not be hardcoded.

File: Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -14,32 +14,43 @@
...
+    <meta name="theme-color" content="#4e73df">
     @RenderSection("Styles", false)
 </head>
 
 <body id="page-top">
 
+    <!--<div id="loader" style="height:100vh">
+        <iframe src="~/img/venjix.svg" style="height:100vh;width:100%">
+        </iframe>
+    </div>-->
```
Comment (fahminlb33) [inline Views/Shared/_Layout.cshtml]: If this is unwanted, please remove it

File: Views/Webhooks/Edit.cshtml [hunk 3]
```diff
@@ -53,7 +54,7 @@
 
                     <h4>How to string format?</h4>
                     <p>The recorded value will be availiable as <b>{{value}}</b> placeholder. Use this placeholder in your URI or body format.</p>
-
+
```
Comment (fahminlb33) [inline Views/Webhooks/Edit.cshtml]: Please remove this empty line

File: Views/Sensors/Edit.cshtml [hunk 4]
```diff
@@ -40,4 +40,8 @@
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+    <div class="col-12 col-sm-12 col-md-5 col-lg-3">
+       
+        
+    </div>
```
Comment (fahminlb33) [inline Views/Sensors/Edit.cshtml]: Is this empty column necessary?

**Review by fahminlb33 — CHANGES_REQUESTED** [review]

File: Views/Home/Index.cshtml [hunk 5]
```diff
@@ -64,34 +62,124 @@
...
+                    </div>
+                </div>
+            </a>
         </div>
     </div>
 
-    <!-- Content Row -->
     <div class="row">
+        <div class="col-xl-4 mb-4">
+            <a class="card lift h-100" href="#!">
```
Comment (fahminlb33) [inline Views/Home/Index.cshtml]: You can point this to Table visualizer

File: Program.cs [hunk 6]
```diff
@@ -56,19 +56,18 @@ public static IHostBuilder CreateHostBuilder(string[] args)
 
         private static void CreateDbIfNotExists(IHost host)
         {
-            using (var scope = host.Services.CreateScope())
+            using var scope = host.Services.CreateScope();
```
Comment (fahminlb33) [inline Program.cs]: Don't have to change this, since this is out of context from your PR

File: Views/Home/Index.cshtml (hunk 1 above)
Comment (rasyidf) [reply Views/Home/Index.cshtml]: For now I'll add the version info to DashboardModel.

File: Views/Shared/_Layout.cshtml (hunk 2 above)
Comment (rasyidf) [reply Views/Shared/_Layout.cshtml]: It's not, I'm just playing with loader or something but it's unnecessarily slow.

File: Views/Sensors/Edit.cshtml (hunk 4 above)
Comment (rasyidf) [reply Views/Sensors/Edit.cshtml]: For now yes, I'm planning to add information later, I'll add comment on that.

File: Views/Home/Index.cshtml (hunk 5 above)
Comment (rasyidf) [reply Views/Home/Index.cshtml]: Consider it done.

File: Program.cs (hunk 6 above)
Comment (rasyidf) [reply Program.cs]: Uh no, I accidentally run my linter.

**Comment by rasyidf** [comment]
> * Sensors > Statistics, the button is too large

I've solved this issued and changed every Header and button into custom component.

> * Forecast (ML), the chart should be taller to allow more precise visualization

It is, I'm developing on wide screen so it is necessary to split the screen. but for responsiveness we might review it until before release.

> * Users, the button for admin is grayed out but still clickable, is the the desired changes? (might be related to #8).

It's just inconsistency of theme, I'll fix that on next push.

> * Settings, please remove the whitespace after status text. Punctuation should come immediately after the text without whitespace.

Done, 

> * About, maybe we should add out LinkedIn too, just in case. Also, would it be possible to use icons instead of text links?

Well, that's minor changes, maybe next time.

File: Models/DashboardModel.cs [hunk 7]
```diff
@@ -7,5 +7,6 @@ public class DashboardModel
         public int SensorsCount { get; set; }
         public long RecordedDataCount { get; set; }
         public DateTime? LastUpdate { get; set; }
+        public string VenjixVersion { get; set; } = System.Reflection.Assembly.GetEntryAssembly().GetName().Version.ToString();
```
Comment (fahminlb33) [inline Models/DashboardModel.cs:10]: Model shouldn't have logic, please create a new method on Helper class (create a new one if one doesn't exists) then move this to controller action.

**Review by fahminlb33 — APPROVED** [review]
Good work.

File: Models/DashboardModel.cs (hunk 7 above)
Comment (fahminlb33) [reply Models/DashboardModel.cs:10]: I'll do this myself.

**Pull request merged**